lucien-ios/StartMyCollection/StartMyCollectionViewController.swift

```diff
@@ -9,9 +9,48 @@
 import UIKit
 
 final class StartMyCollectionViewController: UIViewController {
+
+    @IBOutlet private weak var addBookButton: UIButton!
+    @IBOutlet private weak var welcomeImageView: UIImageView!
+    @IBOutlet private weak var toDashboardButton: UIButton!
+    @IBOutlet private weak var getStartedDescription: UILabel!
+    @IBOutlet private weak var welcomeLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpStyling()
+    }
+
+    func setUpStyling() {
+        welcomeLabel.font = LucienTheme.Fonts.muliExtraLightItalic(size: CGFloat(48.0))
```

Comment:
Each chunk of lines starting with the same label should be grouped into a smaller function called `setup<whatever it's setting up>` e.g. `setupWelcomeLabel`